package.json

```diff
@@ -127,7 +127,7 @@
   "size-limit": [
     {
       "path": "commonjs/index.js",
-      "limit": "70 KB"
+      "limit": "180 KB"
```

Comment:
I'm not sure what this does, @jeroenransijn is this ok?

Reply:
It checks the size of the compiled evergreen package using webpack. Apparently v4 increases the package size by 100kb 😱 

https://github.com/ai/size-limit

Reply:
@jeroenransijn looks like blueprint icons contribute significantly to the build size.